test/integration/targets/win_uri/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+test_uri_path: C:\ansible\win_uri
+httpbin_host: httpbin.org
```

Comment:
So I was told we didn't want to do test against Internet infrastructure, that's why we have this long-standing issue open regarding having access from the Windows VM's to the httptester infrastructure that Docker images are using.

Reply:
We don't, this is a wait and see thing. If we come across any recurring issues we will disable the tests in CI but until we have added the code to forward the httptester container ports to the Windows host this is better than nothing.

Reply:
Fine by me. Although enabling httptester wasn't supposed to be so hard either ;-)

Reply:
We are getting close, there is a PR out there to install the SSH server on the Windows hosts and @mattclay is looking into creating a remote port forwarder to the Windows host with this SSH server for certain tests. We can do all this manually but the trick is getting it to work in CI and be stable.

Once that is in, it will be as simple as changing this variable to be `localhost:port`.

Reply:
Ouch that sounds messy :-\ And wouldn't that be tricky to run from your laptop ?